python/chronos/src/bigdl/chronos/model/tf2/TCN_keras.py

```diff
@@ -197,7 +197,7 @@ def model_creator(config):
                             kernel_size=config.get("kernel_size", 7),
                             dropout=config.get("dropout", 0.2),
                             repo_initialization=config.get("repo_initialization", True))
-    inputs = Input(shape=(config["past_seq_len"], config["input_feature_num"]))
+    inputs = np.zeros(shape=(1, config["past_seq_len"], config["input_feature_num"]))
```

Comment:
so does this mean we only need a "valid" input to go through the model, then the weights are initialized?

Reply:
Yes, using a small dataset with the same shape can speed up initialization compared to Input.